app/controllers/feedback_controller.rb

```diff
@@ -9,6 +9,7 @@ def admin
 
   def new
     @feedback = Feedback.new
+    session[:subject] = params[:subject] || "Dispatch"
```

Comment:
The reason is that "Caseflow" is the name of the whole ecosystem we're building. It's the most generic name for the thing we're working on. Dispatch is one piece of the Caseflow ecosystem. I wouldn't assume that all feedback is for Dispatch, but it all should be about Caseflow.